modules/services/sxhkd.nix

```diff
@@ -22,6 +22,21 @@ in
   options.services.sxhkd = {
     enable = mkEnableOption "simple X hotkey daemon";
 
+    package = mkOption {
+      type = types.package;
+      default = pkgs.sxhkd;
+      defaultText = "pkgs.sxhkd";
+      description = "Package containing the <command>sxhkd</command> executable.";
+    };
+
+    commandline = mkOption {
```

Comment:
We have elsewhere called this `extraOptions`, e.g.
https://github.com/nix-community/home-manager/blob/7e5fee4268f53be8758598b7634dff8b3ad8a22b/modules/services/redshift.nix#L112-L120

So perhaps good to reuse that name here?